### PR TITLE
UIDATIMP-294: Field Mapping Profile details: Order PO & POL from MARC Bib (Create/Edit)

### PR DESCRIPTION
**Pull request by mariia-aloshyna**
## Purpose

To record the details of field mappings and defaults required to create one or more Orders and their Order lines from an imported MARC Bibliographic record file

## Approach

- Implement MappingOrderDetails section
- Provide paths to order fields in ORDER.js file
**NOTE**: fields that were not found in schemas are marked as `// ?`

## Refs

https://issues.folio.org/browse/UIDATIMP-294

## Screenshots

image
image
image
image
image
image
image

### CODE REVIEW

File: src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/EResourcesDetails.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  Row,
+  Col,
+  TextField,
+  Checkbox,
+  Datepicker,
+} from '@folio/stripes/components';
+
+import {
+  AcceptedValuesField,
+  FieldOrganization
```
Comment (OleksandrHladchenko1) [inline src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/EResourcesDetails.js:20]:
```suggestion
  FieldOrganization,
```

File: src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/ItemDetails.js [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+            label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.itemDetails.field.subscriptionFrom`} />}
+            dateFormat="MM-DD-YYYY"
+            name={getFieldName(21)}
+          />
+        </Col>
+        <Col xs={4}>
+          <Field
+            component={Datepicker}
+            label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.itemDetails.field.subscriptionTo`} />}
+            dateFormat="MM-DD-YYYY"
```
Comment (OleksandrHladchenko1) [inline src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/ItemDetails.js:86]:
```suggestion
            dateFormat="YYYY-MM-DD"
```
and add `dateFormat` prop for all the datepickers

File: src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/OrderInformation.js [hunk 3]
```diff
@@ -0,0 +1,425 @@
+import React, {
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  useIntl
```
Comment (OleksandrHladchenko1) [inline src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/OrderInformation.js:9]:
```suggestion
  useIntl,
```

**Review by OleksandrHladchenko1 — CHANGES_REQUESTED** [review]

**Comment by mariia-aloshyna** [comment]
@folio-org/fe-tl-reviewers take a look, please

**Review by OleksandrHladchenko1 — APPROVED** [review]

**Review by Dmitriy-Litvinenko — APPROVED** [review]

File: src/settings/MappingProfiles/initialDetails/ORDER.js [hunk 4]
```diff
@@ -2,18 +2,542 @@ const ORDER = {
   name: 'order',
   recordType: 'ORDER',
   mappingFields: [{
-    name: 'discoverySuppress',
+    name: 'poStatus',
     enabled: true,
-    path: 'order.discoverySuppress',
+    path: 'order.po.poStatus', // ?
```
[the remark this answers is not in the document]
Comment (mariia-aloshyna) [reply src/settings/MappingProfiles/initialDetails/ORDER.js:7]: Described in PR description. Will remove this once @KaterynaSenchenko take a look and we decide what to do with fields that are absent in schemas

**Pull request merged**